docs/reference/twig_helpers.rst

```diff
@@ -15,6 +15,11 @@ Render a page url
     {{ url(page) }} => https://sonata-project.org/absolute/url/to/url
 
     {{ url(page, {}, true) }} => //sonata-project.org/network/path/to/url
+    
+.. note::
+
+    You can use ``Technical Alias`` to link with the pages e.g:
+      ``{{ path('_page_alias_your_page') }}``.
```

Comment:
```suggestion
    
.. note::

    In case you need to use a page's router in twig files, you can use ``pageAlias`` e.g:
      ``{{ path('_page_alias_your_page') }}``.
     You will find this field in your sonata page admin, the ``pageAlias`` is named of ``Technical Alias``.
```

@core23 what do you think it's better or should I change something or put some screenshot.

Reply:
@core23 ?